src/Http/GarminClient/GarminAuthenticator.php

```diff
@@ -54,6 +61,7 @@ public function __construct(
         #[Autowire(env: 'GARMIN_AUTHENTICATION_FILE_PATH')]
         private readonly string $garminAuthenticationFilePath,
     ) {
+        $this->browser = new HttpBrowser($httpClient);
```

Comment:
The `HttpClient` component is stateless and doesn't persist cookies between requests.  I noticed that the MFA submit request fails if the expected cookies are not on the request.  For these requests, I instead used `HttpBrowser` to wrap the client, which is stateful and persists cookies.